Resources/Public/Javascript/PageView/SearchInDocument.js

```diff
@@ -234,15 +256,22 @@ function getNavigationButtons(start, numFound) {
 function getCurrentPage() {
     var page = 1;
     var queryParams = getCurrentQueryParams(getBaseUrl(" "));
+    var pageFound = false;
 
     for(var i = 0; i < queryParams.length; i++) {
         var queryParam = queryParams[i].split('=');
 
         if(decodeURIComponent(queryParam[0]) === $("input[id='tx-dlf-search-in-document-page']").attr('name')) {
             page = parseInt(queryParam[1], 10);
+            pageFound = true;
         }
     }
 
+    if (!pageFound) {
+        var url = baseUrl.split('/');
```

Comment:
Unfortunately, this code fails at this point: 

```
SearchInDocument.js?1640205096:271 Uncaught ReferenceError: baseUrl is not defined
    at getCurrentPage (SearchInDocument.js?1640205096:271)
    at addImageHighlight (SearchInDocument.js?1640205096:286)
    at Object.success (SearchInDocument.js?1640205096:366)
    at c (jquery-3.5.1.min.js?1605185398:2)
    at Object.fireWith [as resolveWith] (jquery-3.5.1.min.js?1605185398:2)
    at l (jquery-3.5.1.min.js?1605185398:2)
    at XMLHttpRequest.<anonymous> (jquery-3.5.1.min.js?1605185398:2)
```

Reply:
If I add  `var baseUrl = getBaseUrl(" ");` above, it doesn't failt. But the highlighting isn't shown.

Reply:
I have added logger for page. Could you tell me for which page highlighting is performed? (I have no test system with configured slugs, so I'm testing with use of fake links and there it looks correct) I was assuming that problem with highlight after link correction comes from incorrect page number taken by highlight method. This page is checked out by getCurrentPage() method.